### PR TITLE
[IMP]convert_field_to_html: manage translated fields

### PR DESCRIPTION
**Pull request by clementelabore**
Improve convert_field_to_html method: added translate=False parameter to handle translatable fields

### CODE REVIEW

**Comment by clementelabore** [comment]
@florian-dacosta

**Review by yvaucher — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review)

**Comment by sebastienbeau** [comment]
@pedrobaeza @legalsylvain maybe this interest you. It replace the following PR : https://github.com/OCA/openupgradelib/pull/358

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -2461,12 +2462,31 @@ def default_func(cr, pool, rec_id, vals):
...
-def convert_field_to_html(cr, table, field_name, html_field_name, verbose=True):
+def convert_field_to_html(
+    cr, table, field_name, html_field_name, verbose=True, translate=False
+):
     """
     Convert field value to HTML value.
 
     .. versionadded:: 7.0
     """
+
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py:2473]: Don't add empty lines inside a method, please (and the same for the next one)

**Comment by clementelabore** [comment]
What are the next step to merge and create a new release usable with pip ?

File: openupgradelib/openupgrade.py (hunk 1 above)
Comment (pedrobaeza) [reply openupgradelib/openupgrade.py:2473]: The empty lines still exist

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**